### DISCUSSION TITLE
http request node 填写了 body，保存完后，还是 none 选项，填写的json丢失！！！

### DISCUSSION THREAD

**Discussion by AngleMAXIN** [langgenius/dify#4118] [Help]
接口都是200  ;https://cloud.dify.ai/console/api/apps/7b3b68ca-f697-4872-9141-24fc455eb972/workflows/draft
![image](https://github.com/langgenius/dify/assets/30991830/b689c7f2-3d03-451e-a229-91d350bd6825)
click again ...
![image](https://github.com/langgenius/dify/assets/30991830/9ba575bc-8883-4f16-8b13-92d1075016e1)

**Comment by crazywoola** [comment]
Close due to #1542
